Last-Update: 2016-01-11
Forwarded: no
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=791805
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: add PostgreSQL 9.5 path
 Perhaps there shall be a better way to find PostgreSQL executable than
 to search through the list of hard-coded paths...

Index: akonadi/src/server/storage/dbconfigpostgresql.cpp
===================================================================
--- akonadi.orig/src/server/storage/dbconfigpostgresql.cpp	2015-09-05 12:43:05.975761742 +0200
+++ akonadi/src/server/storage/dbconfigpostgresql.cpp	2015-09-05 12:45:06.906930014 +0200
@@ -84,7 +84,9 @@
                            << QStringLiteral("/usr/lib/postgresql/9.0/bin")
                            << QStringLiteral("/usr/lib/postgresql/9.1/bin")
                            << QStringLiteral("/usr/lib/postgresql/9.2/bin")
-                           << QStringLiteral("/usr/lib/postgresql/9.3/bin");
+                           << QStringLiteral("/usr/lib/postgresql/9.3/bin")
+                           << QStringLiteral("/usr/lib/postgresql/9.4/bin")
+                           << QStringLiteral("/usr/lib/postgresql/9.5/bin");
 
         defaultServerPath = XdgBaseDirs::findExecutableFile(QStringLiteral("pg_ctl"), postgresSearchPath);
         defaultInitDbPath = XdgBaseDirs::findExecutableFile(QStringLiteral("initdb"), postgresSearchPath);
